### ISSUE TITLE
On Platform.sh this requires server_variable configuration for Doctrine

### ISSUE THREAD

**Issue by janit** [ezsystems/ezcommerce#4]
Due to unknown reasons (to me), installation to Platform.sh / eZ Platform Cloud you need to define the `server_version` variable for Doctrine to `mariadb-10.2.12`. More information in this ticket: https://github.com/doctrine/DoctrineBundle/issues/673

Example configuration (in `app/config/config.yaml`):

```
doctrine:
    dbal:
        connections:
            default:
                driver: '%database_driver%'
                host: '%database_host%'
                port: '%database_port%'
                dbname: '%database_name%'
                user: '%database_user%'
                password: '%database_password%'
                charset: utf8mb4
                server_version: mariadb-10.2.12
```

**Comment by andrerom** [comment]
@Crell Possible to detect this by info in `PLATFORM_RELATIONSHIPS ` or otherwise? So we can dynamically set it in:
https://github.com/ezsystems/ezplatform/blob/master/app/config/env/platformsh.php

**Comment by Crell** [comment]
This is a long standing flaw in Doctrine that they've opted to not fix. We've not had that exposed in the relationship, which is why we've hard coded it.

However, we *just* (as in, the ticket was only marked closed today) added support for exposing the server version number in the relationships array.  I'll be adding it to the public documentation soon-ish.

On the new regions only (us2, eu2, au), there should be a `type` property in the relationship that is the same as the `type` key of the service in `services.yaml`.  So it will look like "mysql:10.2" or similar.  It should be trivial enough to parse out of that in PHP to populate `server_version` dynamically.

**Comment by andrerom** [comment]
@crell and how long will it take for this to be on all regions? And PE... 🤷‍♂️

**Comment by Crell** [comment]
It's on the new regions now, I believe.  (us2, eu2, au).  It will never be on the older us and eu regions, as they're running the old v1 of our coordination software that is in maintenance only.  However, you cannot create new projects on those regions anymore so that's not an issue for any future customers.

For PE, it will involve adding information to the static relationship configuration for the cluster.  We have not yet updated the automation scripts to do that automatically when a project is created, but we can add that one-off on individual projects on request until that is added.

**Comment by andrerom** [comment]
Hmm, sounds like we'll pretty soon need document our config is for new regions and not old (things like this, varnish, ..), but ok..

**Comment by Crell** [comment]
Users on old regions can migrate to new regions fairly easily.  I'd recommend it, as the new regions should be faster, too, aside from being where much new functionality will be.  (Some things will apply to all regions, others just to new ones, depending on what part of the system they're in.)

**Comment by andrerom** [comment]
Kind of fixed by @vidarl in #5

Follow up here is to take advantage of more dynamic way to get this and apply this also on ezplatform as anyone using ORM will hit this.
